### PR TITLE
Start create questions tests.

### PR DESCRIPTION
**Pull request by mbconsalting**

### CODE REVIEW

**Comment by nadia39** [comment]
Hi, I would like to contribute to this issue.
I need to contribute to open source projects for class.
What questions were you thinking of adding to your project?

**Comment by mbconsalting** [comment]
Hi, Nadia. It's not open source project, i learning in thinknetica IT-school. It's my lesson, i must find solution by myself. Thank you for assistance.

File: app/controllers/api/v1/profiles_controller.rb [hunk 1]
```diff
@@ -1,19 +1,12 @@
...
+class Api::V1::ProfilesController < Api::V1::BaseController
+  authorize_resource class: User
 
   def me
     respond_with current_resource_owner
   end
 
   def index
-    respond_with User.where.not(id: current_resource_owner)
+    respond_with User.without(current_resource_owner)
```
Comment (psylone) [inline app/controllers/api/v1/profiles_controller.rb]: ❤️

File: spec/api/v1/answers/show_spec.rb [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+      it 'returns 401 status if access_token is invalid' do
+        get "/api/v1/questions/#{question.id}/answers/#{answer.id}", params: { format: :json, access_token: '1234' }
+        expect(response.status).to eq 401
+      end
+    end
+
+    context 'authorized' do
+      before do
+        get "/api/v1/questions/#{question.id}/answers/#{answer.id}", params: { format: :json, access_token: access_token.token }
```
Comment (psylone) [inline spec/api/v1/answers/show_spec.rb]: Если я всё верно понимаю, в [действии контроллера](https://github.com/mbconsalting/stackforum/pull/14/files#diff-93bda3ff6fc6fa27f1ff046ca6b75cafR11) мы не используем объект вопроса при поиска ответа.

**Pull request merged**